### DISCUSSION TITLE
64-bit Application?

### DISCUSSION THREAD

**Discussion by ghost** [mienaiyami/yomikiru#21] [Q&A] [answered]
**64-bit Application** 

- The application is 32-bit and It works on both 32-bit and 64-bit systems.
- But the 64-bit version will run faster and generally perform better than the 32-bit one if it is well designed. Am I wrong?
- Is it harder to design the application for 64-bit?

I do not know about coding or programming, but I think the difference will be like similar to 'native 4K' vs 'upscaled 4K from 1080p'.

**Comment by mienaiyami** [comment, marked as answer]
According to what I know, only difference in 32-bit and 64-bit is memory allowed to application. Originally this app used to be 64bit but then made it 32bit since there was no performance difference, I'm leaving link of a portable version of 64bit if you want to test. https://mega.nz/file/gAwRQaqL#w9WB1rlWxkw-dD-vQ6yBmX8PkefkeJt-CGiHS_6iPZE

As for why i didn't make both 32bit and 64bit?
I wasn't able to automate compiling and uploading of apps on GitHub. So, you can imagine compiling 4 difference versions and then uploading them and not to mention it also create some problem is auto update.

**Reply by ghost** [reply]
Yes, there is no performance difference.
